Automattic/wp-calypso
# Jetpack Manage: Fixing discount pricing display for products with prices higher than 1000

Related to https://github.com/Automattic/jetpack-manage/issues/319

## Proposed Changes

* This PR fixes an issue within the existing code that resulted in product (bundle) amounts greater than 1,000 being severely discounted. This was due to the use of parseFloat on a string amount, which for amounts higher than 1,000 included a comma.


## Testing Instructions


* To test the original issue, visit the Partner Portal Issue Licenses page in trunk, selecting a bundle size of 50 or 100 (eg. `http://jetpack.cloud.localhost:3000/partner-portal/issue-license?bundle_size=100`). You will see a severely reduced price for the Jetpack VaultPress Backup Add-on of 5TB (and 3TB for a bundle size of 100), plus a couple of other products.
* Then after applying this PR, visit the same page and bundle sizes, and the prices showing should be correct (greater than 1,000).

Screenshots:

What you should now see for a bundle size of 100 (for Backup products):
<img width="1032" alt="Screenshot 2024-02-22 at 15 27 06" src="https://github.com/Automattic/wp-calypso/assets/16754605/0906fb74-8b10-409f-a08d-d9ae0bdfde24">


## Pre-merge Checklist
